### PR TITLE
Ansible 5.2.0: Dependencies, changelog and porting guide

### PR DESCRIPTION
**Pull request by dmsimard**

### CODE REVIEW

File: 5/ansible-5.2.0.deps [hunk 1]
```diff
@@ -0,0 +1,101 @@
+_ansible_version: 5.2.0
+_ansible_base_version: 2.12.0
```
Comment (dmsimard) [inline 5/ansible-5.2.0.deps:2]: Needs to be rebuilt after modifying this manually with 2.12.1.

What bug was this again ?

**Review by felixfontein — APPROVED** [review]

**Comment by dmsimard** [comment]
The failure for the pre-commit.ci job is unrelated.

Thanks for reviewing @felixfontein !

**Comment by felixfontein** [comment]
Is there a reason that pre-commit.ci is active for this repo?

**Comment by dmsimard** [comment]
> Is there a reason that pre-commit.ci is active for this repo?

@felixfontein I am not the one who enabled it but it was installed in the ansible-community org and it appears to have been applied to every repository...

**Pull request merged**